### DISCUSSION TITLE
Multi-word shortcut names don't appear to run successfully

### DISCUSSION THREAD

**Discussion by RosemaryOrchard** [ttscoff/bunch-docs#274] [General]
I'm wondering if this is something that is unique to me, as I tried what the docs pages say and it didn't work 🤪 

If my Bunch includes:
```sh
(sc My Shortcut Name)
```
And I have a shortcut with the name `My Shortcut Name`, I find that the shortcut will not be run and the log will say that the shortcut `My` could not be run.

I've worked around this by changing my actions to:
```sh
(sc "My Shortcut name")
```

That's fine, that seems to work, but the documentation has numerous examples without quotation marks with multi word names. So have I broken my Shortcuts somehow? 😁

**Comment by ttscoff** [comment]
It should work without the quotes, I’ll look into it.

- Brett

**Comment by ttscoff** [comment]
Just pushed beta build 170, assuming you're running the beta, please 
update and let me know if it solves the issue.

-Brett

**Comment by RosemaryOrchard** [comment]
Sorry for the very delayed reply—it works perfectly now!
